semantic/src/themes/universe/collections/form.overrides

```diff
@@ -90,7 +90,10 @@
 ---------------------*/
 
 .ui.form .fields .field > label ~ small {
-  font-size: 12px;
+  font-size: 12px !important;
+  display: inline-block !important;
+  margin-bottom: 8px !important;
+  color: grey !important;
```

Comment:
is !important required when you're doing this in the overrides file?

Reply:
from what I've experienced, yep :\

Reply:
I imagine there are a few cases where it's not required, though.